sql-api/src/main/java/io/cdap/pipeline/sql/api/StructuredQuery.java

```diff
@@ -70,7 +70,7 @@ public String getAlias() {
   }
 
   @Override
-  public QueryableType getSourceType() {
+  public QueryableType getQueryableType() {
```

Comment:
I think it's better if you just have a getFrom() method, and force the caller to call getFrom().getType()

Reply:
Fixed, but leaving in the getType() method since StructuredQuery can be selecting from another StructuredQuery, so StructuredQuery implements Queryable.